man/mustache.5.ron

```diff
@@ -411,6 +411,37 @@ Can be thought of as a single, expanded template:
     {{/names}}
 
 
+**Dynamic Names**
+
+Partials can be loaded dynamically at runtime using Dynamic Names; an
+**optional** special notation which allows to dynamically determine a tag's
```

Comment:
"optional" is a bit ambiguous here. It can be interpreted as "optional part of the spec" but also as "optional to use". I suggest adding some words to clarify that not every Mustache engine might implement this notation.